lib/__tests__/test.js

```diff
@@ -2,7 +2,40 @@
 
 const fs = require('fs');
 const path = require('path');
-const defineTest = require('jscodeshift/src/testUtils').defineTest;
+const {walkDir} = require('../execution-helper');
+const rewriter = require('../rewriter');
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} filePrefix - the test file prefix
+ */
+function defineTest(dirname, filePrefix) {
+  const testName = `rewrites correctly on ${filePrefix}`;
+  describe('rewriter', () => {
+    it(testName, () => {
+      runTest(dirname, filePrefix);
+    });
+  });
+}
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} filePrefix - the test file prefix
+ */
+function runTest(dirname, filePrefix) {
```

Comment:
It'd be better if this function only took a filePrefix. To do that, you could replace the first line with

```js
path.resolve(filePrefix, '..', '__testfixtures')
```